rdmo/core/permissions.py

```diff
@@ -74,3 +76,23 @@ def has_permission(self, request, view):
             return True
         else:
             return False
+
+
+class CanToggleElementCurrentSite(DjangoModelPermissions):
+
+    perms_map = {
+        'PUT': ['%(app_label)s.change_%(model_name)s_toggle_site'],
+    }
+
+    @log_result
+    def has_permission(self, request, view):
+
+        try:
```

Comment:
I don't think this belongs here, this is not a permission check. I would just remove the method and use the one from the parent model.

Reply:
Ive kept:
```
    @log_result
    def has_permission(self, request, view):
        return super().has_permission(request, view)
```
otherwise there is no logging? 
It tests the rule ` 'PUT': ['%(app_label)s.change_%(model_name)s_toggle_site'],`  right?